### DISCUSSION TITLE
Backup and restore root pool (rpool) of ubuntu installation

### DISCUSSION THREAD

**Discussion by SimonBard** [openzfs/zfs#16257] [General]
There are similiar topics here, but I guess its only about data, not the system installation.

My SSD recently broke down and its not even recognized anymore by anything (not as external or internal drive). I have a backup of course, one is on a external USB drive. When I installed the system, ubuntu created a bpool and rpool. I backupped the rpool. From what I know, it contains the system installation and all the data.

I have now created a zpool with a HDD and a cache. Now I want to copy the old backup to the newly created system. How can I perform this?

I have read [this](https://www.reddit.com/r/zfs/comments/lnisu9/easiest_way_to_clonemigrate_a_pool/):
syncoid -r oldpool newpool/oldpool

But will my system then start normally as it is now in "newpool/oldpool", so in a child dataset instead of the root dataset? Do I just have to make sure its mounted to the root?

When I try to boot from the external HDD, I get many mount errors and system will not start.

**Comment by no-usernames-left** [comment]
Hi Simon,

Your requests are probably a better fit over at https://discourse.practicalzfs.com/
